components/Starknet/modules/ROOT/pages/index.adoc

```diff
@@ -1,7 +1,189 @@
 [id="overview"]
 = Overview
 
-Starknet is a permissionless, decentralized validity rollup, also known as a zero-knowledge rollup (ZK rollup). It operates as a Layer 2 (L2) network over Ethereum, enabling any dApp to achieve unlimited scale for its computation - without
-compromising Ethereum’s composability and security - thanks to Starknet’s reliance on the safest and most scalable cryptographic proof system: STARK.
+Starknet is a permissionless, decentralized zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2), enabling any dApp to achieve unlimited scale for its computation without compromising on Ethereum’s composability
+and security.
 
-Starknet contracts and the Starknet OS are written in Cairo – supporting the deployment and scaling of any use case, whatever the business logic.
+Starknet achieves low cost transactions and high performance by using its own cryptographic proof system called STARK. Starknet contracts and the Starknet OS are written in link:https://github.com/starkware-libs/cairo[Cairo], a custom-built and
```

Comment:
```suggestion
Starknet achieves secure low cost transactions and high performance by using the STARK cryptographic proof system. Starknet contracts and the Starknet OS are written in link:https://github.com/starkware-libs/cairo[Cairo], a custom-built and
```

I'm not sure that it's correct to say that STARKs are Starknet's own system. Other chains can use them as well, not to mention StarkEx.